cardano-cli/test/cardano-cli-golden/files/golden/help/conway_governance_action_create-update-proposal.cli

```diff
@@ -0,0 +1,130 @@
+Usage: cardano-cli conway governance action create-update-proposal 
+                                                                     [--protocol-major-version NATURAL
+                                                                       --protocol-minor-version NATURAL]
+                                                                     [--decentralization-parameter RATIONAL]
```

Comment:
[--decentralization-parameter RATIONAL] <-- This no longer exists in Conway

Reply:
Also, @Jimbo4350 `create-update-proposal` is a bit vague, it might be `create-protocol-parameter-change`, `create-protocol-parameter-proposal` which is closer to how it is called on the CIP.